src/App.tsx

```diff
@@ -1,61 +1,147 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types';
+
+const getUserById = (userId: number) => {
+  return usersFromServer.find(user => user.id === userId)
+  || usersFromServer;
+};
```

Comment:
```suggestion
const getUserById = (userId: number) => {
  return usersFromServer.find(user => user.id === userId)
  || null;
};
```
better to return null instead of all array If the user wasn’t found